include/treelite/c_api.h

```diff
@@ -171,6 +171,28 @@ TREELITE_DLL int TreeliteLoadXGBoostJSONString(const char* json_str,
 TREELITE_DLL int TreeliteLoadXGBoostModelFromMemoryBuffer(const void* buf,
                                                           size_t len,
                                                           ModelHandle* out);
+
+TREELITE_DLL int TreeliteLoadSKLearnRandomForestRegressor(
```

Comment:
Consider adding documentation that this method supports ExtraTrees as well.

Reply:
Got it. I will add docstrings to all functions introduced in this pull request.